### PR TITLE
go.mod: better go-ordered-json

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Almost no functional changes, the most important one is that virtuald dependency is removed.

### CODE REVIEW

**Review by roman-khimov — APPROVED** [review]
Huh. It allows you to build this? I though it shouldn't work with 1.19, but maybe it got more strict about it in later versions. I expected to update this with 1.20 update, but if it works then OK.

**Comment by AnnaShaleva** [comment]
>  I though it shouldn't work with 1.19

Let's wait for Go update before the merge.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**
